Dockerfile

```diff
@@ -83,17 +83,18 @@ COPY --from=js-builder /usr/src/app/public ./public
 COPY --from=js-builder /usr/src/app/tools ./tools
 
 # LOGZ.IO GRAFANA CHANGE :: Copy custom.ini
+COPY custom.ini conf/custom.ini
 RUN cp "$GF_PATHS_HOME/conf/custom.ini" "$GF_PATHS_CONFIG"
 # LOGZ.IO GRAFANA CHANGE :: Preinstall plugins
-COPY ./data/plugins "$GF_PATHS_PLUGINS"
+COPY grafana/data/plugins "$GF_PATHS_PLUGINS"
 # LOGZ.IO GRAFANA CHANGE :: Remove news panel
 RUN rm -rf ./public/app/plugins/panel/news
 # LOGZ.IO GRAFANA CHANGE :: Remove pluginlist panel
 RUN rm -rf ./public/app/plugins/panel/pluginlist
```

Comment:
Should be `grafana/public/app/plugins/panel/pluginlist`

Reply:
no, this is the correct path. we use these plugins https://github.com/logzio/data-viz/tree/logzio-v8.0.3/data/plugins